doc/nrf/device_guides/nrf91/nrf91_snippet.rst

```diff
@@ -42,46 +46,50 @@ To enable modem traces with the flash backend, use the following command, where
 nRF91 modem tracing with UART backend using snippets
 ****************************************************
 
-Snippet enables the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE` Kconfig option and chooses the Zephyr UART driver for the backend, with the necessary Kconfig options.
+The ``nrf91-modem-trace-uart`` snippet enables the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE` Kconfig option and chooses the Zephyr UART driver for the backend, with the necessary Kconfig options.
 The snippet also enables the UART1 peripheral with a baud rate of 1 Mbd and hardware flow control enabled.
 If this configuration does not match your requirements, you can add a snippet or Kconfig and devicetree overlays to your application with the desired setup.
+
+.. note::
+    If you are using the nRF9160 DK, remember to :ref:`set the board controller switch to the **nRF91** position <build_pgm_nrf9160_board_controller>` before programming.
+
 To enable modem tracing with the UART trace backend on a nRF91 device, add the ``nrf91-modem-trace-uart`` snippet to the :term:`build configuration`.
 This can be done in one of the following ways:
 
-With west
-=========
+.. tabs::
 
-To add the modem trace UART snippet when building an application with west, use the following command, where *board_target* corresponds to your development kit board target and `<image_name>` to your application image name:
+   .. group-tab:: west
 
-.. parsed-literal::
-   :class: highlight
+      To add the modem trace UART snippet when building an application with west, use the following command pattern, where *board_target* corresponds to your development kit board target and `<image_name>` to your application image name:
 
-   west build --board *board_target* -- -D<image_name>_SNIPPET="nrf91-modem-trace-uart"
+      .. parsed-literal::
+        :class: highlight
 
-.. note::
-   With :ref:`sysbuild <configuration_system_overview_sysbuild>`, using the ``west build -S`` option applies the snippet to all images.
-   Therefore, use the CMake argument instead, specifying the application image.
+        west build --board *board_target* -- -D<image_name>_SNIPPET="nrf91-modem-trace-uart"
+
+      .. note::
+        With :ref:`sysbuild <configuration_system_overview_sysbuild>`, using the ``west build -S`` option applies the snippet to all images.
+        Therefore, use the CMake argument instead, specifying the application image.
```

Comment:
```suggestion
         With :ref:`sysbuild <configuration_system_overview_sysbuild>`, using the ``west build -S`` option applies the snippet to all images.
         Therefore, use the CMake argument instead, for specifying the application image.
```

Reply:
I'm not sure whether I understand this second sentence. Use CMake argument instead of what? Also the subclause after the comma is also a bit odd. Would it need something more, like when specifying or for specifying.